packages/client3/pages/artist/pool/[address]/index.page.tsx

```diff
@@ -1,42 +1,56 @@
-import axios from "axios";
+import { useQuery } from "@apollo/client";
+import { BigNumber } from "ethers";
 import { useRouter } from "next/router";
-import { useEffect, useState } from "react";
 
-import { Button, Heading, LinkButton } from "@/components/design-system";
-import {
-  useLayoutContext,
-  useLayoutTitle,
-  useSelectedSidebarItem,
-} from "@/hooks/sidebar-hooks";
+import { Button, Heading } from "@/components/design-system";
+import { useLayoutContext } from "@/hooks/sidebar-hooks";
+import { cryptoToFloat } from "@/lib/format";
+import { SupportedCrypto } from "@/lib/graphql/generated";
+import { backerPool } from "@/queries/backer.queries";
 
 import PoolDetail from "./pool-details";
 import PoolDetailsRightGrid from "./pool-details-grid-right";
 
 function ArtistPoolPage() {
   const { title } = useLayoutContext();
-  const [poolData, setPoolData] = useState<any>(undefined);
   const router = useRouter();
   const { address } = router.query;
-  /**
-   * Below code(useEffect) and it's usages need to be replaced by openTranchedPools when create pool insertion is done
-   */
-  useEffect(() => {
-    const fetchData = async () => {
-      const response = await axios.get(`/api/pool`);
-      const mappedData = Object.keys(response.data).map((key) => ({
-        id: key,
-        ...response.data[key],
-      }));
-      const data = mappedData.find((poolData) => poolData.id === address);
-      setPoolData(data ?? null);
-    };
-    fetchData();
-  }, [address]);
 
-  if (poolData === undefined) {
-    return null;
+  const { data, error, loading } = useQuery(backerPool, {
+    variables: {
+      address: (address as string) + "-2",
+      fetchPolicy: "network-only",
+    },
+  });
```

Comment:
Fetching data for the pool details screen was done as part of FAD-130. Can you please take a look at it to see if something is missing there

Reply:
Same for the components used in this page